torch/csrc/jit/runtime/interpreter/code_impl.h

```diff
@@ -759,8 +766,32 @@ struct CodeImpl {
     }
 
     operator_table_.emplace_back(oper);
+#ifndef NDEBUG
+    full_operator_table_.emplace_back(op);
+#endif
     return size;
   }
+
+  void assert_stack_size(
```

Comment:
Maybe add inline here, if this didn't get inlined then we would be adding overhead in the non-debug. also `inline` doesn't actually guarantee something gets inlined so maybe its better to be safe and add the macros